frame/staking/src/lib.rs

```diff
@@ -1835,3 +2299,49 @@ impl<T, Reporter, Offender, R, O> ReportOffence<Reporter, Offender, O>
 		}
 	}
 }
+
+#[allow(deprecated)]
+impl<T: Trait> frame_support::unsigned::ValidateUnsigned for Module<T> {
+	type Call = Call<T>;
+	fn validate_unsigned(call: &Self::Call) -> TransactionValidity {
+		if let Call::submit_election_solution_unsigned(
+			winners,
+			compact,
+			validator_index,
+			signature,
+		) = call {
+			// TODO: since unsigned is only for validators, and it is not increasing the block
```

Comment:
yes, I don't see a point in verifying a hundred and something unsigned transactions from out hundred and something validators, while we know they will all be the same. I think this is safe to argue since the offchain code is also stored on chain. If a validator has a better solution, then they can submit it separately via signed transaction. 

Although a second solution from a validator will be rejected rather quickly as it will have the same score, but still, that is a lot of processing that goes into the chain for no reason.

Reply:
NO -- then someone can just submit a very very bad solution and then everything that a validator submits will not be included. 

I still think this condition must exists though, but need to be more explicit: If a solution exists, and it is from a validator, then short circuit this one.